### PR TITLE
Add examples of copy which requires variants

### PR DESCRIPTION
**Pull request by stasm**
A few examples of copy for #93.

### CODE REVIEW

**Comment by stasm** [comment]
I added a few examples of English copy with corresponding Polish translations. I think it would be good to add a few more languages here: perhaps French, Finnish and Chinese?

**Comment by nbouvrette** [comment]
For French the examples would look very similar to Polish but if you want I can add them.

I also like the examples, because they are probably the most common ones which would require variants.

**Comment by filmil** [comment]
HI folks.  @zbraniecki pointed me to this workgroup, and I hoped to contribute a bit.

I would like to contribute examples in at least one language customarily written in Cyrillic.  I think it would bring in a few unique real use cases that such languages have, such as inflection (declension and conjugation), and transliteration (`John` becomes `Џон`).

Will wait for this to be merged first, though.

File: exploration/variants.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Regular plurals
+
+### English
+
+    No items selected.
+    1 item selected.
+    2 items selected.
+    5 items selected.
+
```
Comment (eemeli) [inline exploration/variants.md:14]:
```suggestion

### Finnish

    Ei valittuja asioita.
    1 valittu asia.
    2 valittua asiaa.
    5 valittua asiaa.

```

File: exploration/variants.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+## Plurals optional in English
+
+_item_ is skipped for brevity but can be assumed to be the subject.
+
+### English
+
+    1 selected
+    2 selected
+    5 selected
+
```
Comment (eemeli) [inline exploration/variants.md:32]:
```suggestion

### Finnish

    1 valinta
    2 valintaa
    5 valintaa

```

This one's a bit tricky in Finnish; this form refers to "selections" instead.

File: exploration/variants.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    2 zaznaczone
+    5 zaznaczonych
+
+## Gender
+
+### English
+
+    Anne published a post in Birthday Party
+    John published a post in Birthday Party
+
```
Comment (eemeli) [inline exploration/variants.md:45]:
```suggestion

### Finnish

    Anne julkaisi viestin Birthday Party-ryhmässä
    John julkaisi viestin Birthday Party-ryhmässä

```

Presuming from the Polish that "Birthday Party" refers to a group.

File: exploration/variants.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    John opublikował post w grupie Birthday Party
+
+## Gender and Plurals
+
+### English
+
+    Anne and John published a post in Birthday Party
+    Anne and Mary published a post in Birthday Party
+    John and Mark published a post in Birthday Party
+
```
Comment (eemeli) [inline exploration/variants.md:58]:
```suggestion

### Finnish

    Anne ja John julkaisivat viestin Birthday Party-ryhmässä
    Anne ja Mary julkaisivat viestin Birthday Party-ryhmässä
    John ja Mark julkaisivat viestin Birthday Party-ryhmässä

```

**Comment by zbraniecki** [comment]
In https://github.com/zbraniecki/message-format-2.0-rs/pull/7 I attempted at implementing an AST that can handle this examples.

One thing I noticed while doing it is that the examples show only a single selector. The last one is a gender selector over plural, but its still a single selector.
Would we want an example that requires two selectors, for example `Anne published 2 pictures`?

**Comment by aphillips** [comment]
@zbraniecki I think so: nested selection is a common enough feature and managing the complexity is an important consideration.

**Comment by rxaviers** [comment]
Gender... So far we have a Polish example ([maleTaints](https://github.com/unicode-cldr/cldr-core/blob/2c2d91fa0c581b7f3a35fc748a1ce88e070ac550/supplemental/gender.json#L22)). We could add more examples, one for `mixedNeutral` (e.g., Greek), and another for `neutral` (e.g., German).

**Comment by romulocintra** [comment]
Going to merge this one as base case to start building CI pipelines for testing

**Pull request merged**